mentorship ios/Members/SupportingViews/MemberDetailCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MemberDetailCell.swift
+//  mentorship ios
+//
+//  Created by Yugantar Jain on 09/06/20.
+//  Copyright © 2020 Yugantar Jain. All rights reserved.
+//
+
+import SwiftUI
+
+struct MemberDetailCell: View {
+    let title: String
+    let value: String?
+
+    var body: some View {
+        if !(value?.isEmpty ?? true) {
```

Comment:
guard might be a better practice. Early return in case view is empty.

Reply:
okay